import-scripts/import-msk-postimport-steps-for-clickhouse.sh

```diff
@@ -67,10 +68,36 @@ function copy_target_mysql_database_to_target_clickhouse_database() {
     return 0
 }
 
+function download_derived_table_construction_script() {
+    # Check if derived table sql script dirpath exists
+    # If not, try to create it
+    derived_table_sql_script_dirpath="$CREATE_DERIVED_TABLES_SQL_FILE_DIRPATH"
+    if ! [ -e "$derived_table_sql_script_dirpath" ] ; then
+        if ! mkdir -p "$derived_table_sql_script_dirpath" ; then
+            echo "Error: could not create target directory '$derived_table_sql_script_dirpath'" >&2
+            return 1
+        fi
+    fi
+    # Remove any scripts currently in the derived table sql script dirpath
+    if [[ -d "$derived_table_sql_script_dirpath" && "$derived_table_sql_script_dirpath" != "/" ]]; then
+        rm -rf "$derived_table_sql_script_dirpath"/*
+    fi
+    # Attempt to download the derived table SQL files from github
+    if ! $DOWNLOAD_DERVIED_TABLE_SQL_FILES_SCRIPT_FILEPATH --github_branch_name "$CLICKHOUSE_SCHEMA_BRANCH_NAME" "$derived_table_sql_script_dirpath" ; then
+        echo "Error during download of derived table construction .sql files from github" >&2
+        return 1
```

Comment:
I changed this from `exit 1` to `return 1`

Reply:
good catch .. 👍